Directory.Build.props

```diff
@@ -20,4 +20,7 @@
   <ItemGroup>
     <InternalsVisibleTo Include="CommunityToolkit.Maui.UnitTests" />
   </ItemGroup>
+  <PropertyGroup Condition="'$(TF_BUILD)' == 'true' and $([MSBuild]::IsOSPlatform('windows')) == 'true'">
```

Comment:
Can you explain this condition? What's `TF_BUILD`?

Reply:
`TF_BUILD` means run on azure-pipelines 

>Tizen platform include on TargetFrameworks when only if current os platform is windows and it run on CI(TF_BUILD)

https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml
TF_BUILD | Set to True if the script is being run by a build task.  This variable is agent-scoped, and can be used as an environment  variable in a script and as a parameter in a build task, but not as part  of the build number or as a version control tag.
-- | --